### DISCUSSION TITLE
PullRequest REST API mergeable: true always when auto-merge enabled

### DISCUSSION THREAD

**Discussion by taylor-cedar** [community/community#44148] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The following API `/repos/{owner}/{repo}/pulls/{pull_number}` returns 

mergeable: true
mergeable_state: "blocked"

always when the pull request is mergeable or when it's not mergeable when auto-merge setting is turned on. There must be something with auto-merge that causes the values to be broken. This appears to be a bug.

**Comment by taylor-cedar** [comment]
This seems to be an issue on a single repo. Not applicable to all repos

**Comment by fclairamb** [comment]
This is most probably because you have some branch protection rules. So the branch is technically _mergeable_ but one of the branch protection rules puts it in a blocking status.
